server/service/osquery_utils/queries.go

```diff
@@ -646,15 +646,23 @@ func directIngestMDM(ctx context.Context, logger log.Logger, host *fleet.Host, d
 		logger.Log("component", "service", "method", "ingestMDM", "warn",
 			fmt.Sprintf("mdm expected single result got %d", len(rows)))
 	}
-
-	installedFromDep, err := strconv.ParseBool(rows[0]["installed_from_dep"])
```

Comment:
Hm, so you get a row but it's empty?

Reply:
I get a row, with only `enabled: false`, the remaining columns are empty (my host doesn't have MDM configured).

Reply:
Oh, but you're running orbit, right? So you have the table.

Reply:
Correct.

Reply:
Makes sense, I missed that case.